docker-make.yml

```diff
@@ -1,15 +1,19 @@
 ---
 version: '1'
 name: metalctl
-username: metal
-registry-host: registry.fi-ts.io
-default-build-args:
-  - http_proxy=${HTTP_PROXY}
-  - https_proxy=${HTTP_PROXY}
-  - HTTP_PROXY=${HTTP_PROXY}
-  - HTTPS_PROXY=${HTTP_PROXY}
-  - NO_PROXY=${NO_PROXY}
-  - no_proxy=${NO_PROXY}
+username: metalstack
+registry-host: docker.io
+after:
+  - mkdir -p tmp
+  - mkdir -p result
+  # the second latest built container is the builder, which contains the platform binaries
+  - docker cp $(docker create $(docker image ls --format '{{ .ID }}' | sed -n 2p)):/work/bin tmp
```

Comment:
hu, what is this

Reply:
That's copying the binaries on to the Github runner.

Reply:
Do you have a better idea? We need to somehow get the artifacts from the builder container.

Reply:
We can handle this in a different PR. This is the first version that kind of works.